js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -624,34 +633,7 @@ function _buildServiceBundleFromScript (tempServicePath, tempBundlePath, jsBuffe
   utils.zipDirectory(tempServicePath, tempBundlePath);
 }
 
-function _deleteTempPaths({tempServicePath, tempBundlePath}) {
```

Comment:
I've inlined these two functions because they're used in only one place and having them separate just led to playing hot potato with various opaque arguments. Might want to refactor this more sensibly later.